docs: fix 8.3 typo and adding PHP version strategy (#6721)
ddev/ddev: docs/content/users/extend/customization-extendibility.md

## Changing PHP Version

> The project's `.ddev/config.yaml` file defines the PHP version to use. The [`php_version`](../configuration/config.md#php_version) can be `5.6`, `7.0`, `7.1`, `7.2`,  `7.3`, `7.4`, `8.0`, `8.1`, `8.2`, `8.3`, or `8.4`.

Reviewer: ```suggestion
The project's `.ddev/config.yaml` file defines the PHP version to use. The [`php_version`](../configuration/config.md#php_version) can be `5.6` through `8.4`, and new versions are added when they are released by the PHP Foundation.
```

## Changing PHP Version

> 

Reviewer: ```suggestion
```

## Changing PHP Version

> DDEV keeps up with the "more stable" release, which is one version behind the latest.

Author: I agree, and why I asked:

> Also, we could add the DDEV PHP version strategy? **Maybe it belongs somewhere else?**

But I do think it would be valuable to include it somewhere in the documentation. If you agree, the question is where then?

> The project's `.ddev/config.yaml` file defines the PHP version to use. The [`php_version`](../configuration/config.md#php_version) can be changed to `5.6`, `7.0`, `7.1`, `7.2`,  `7.3`, `7.4`, `8.0`, `8.1`, `8.2`, 8.3`, or `8.4`.

Reviewer 2: Thank you, and small suggestion:

```suggestion
The project's `.ddev/config.yaml` file defines the PHP version to use. The [`php_version`](../configuration/config.md#php_version) can be changed to `5.6`, `7.0`, `7.1`, `7.2`,  `7.3`, `7.4`, `8.0`, `8.1`, `8.2`, `8.3`, or `8.4`.
```
